src/lib/gossip_net/gossip_net.ml

```diff
@@ -209,6 +211,7 @@ module Make (Message : Message_intf) :
              (fun client reader writer ->
                Rpc.Connection.server_with_close reader writer ~implementations
                  ~connection_state:(fun _ ->
+                   (* client IP and an ephemeral port, not its discovery or communication port *)
```

Comment:
This comment is a bit confusing, can you expand on this?

Reply:
expanded